cmake/FindRust.cmake

```diff
@@ -142,6 +142,10 @@ if(NOT DEFINED CARGO_HOME)
     endif()
 endif()
 
+#
+# Environment for most invocations of `cargo` 
+#
```

Comment:
I'm not really sure what this code block means to say

Reply:
It's a comment that got detached from its code.